backend/internal/config/init.go

```diff
@@ -0,0 +1,108 @@
+package config
+
+import (
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/cilium/cilium/pkg/crypto/certloader"
+	"github.com/cilium/hubble-ui/backend/internal/msg"
+	"github.com/cilium/hubble-ui/backend/pkg/logger"
+)
+
+const (
+	UIServerDefaultPort = "8090"
+)
+
+func Init() (*Config, error) {
+	cfg := &Config{}
+
+	setupRelayAddr(cfg)
+	setupUIServerPort(cfg)
+
+	if err := setupTLSOptions(cfg); err != nil {
+		return nil, err
+	}
+
+	return cfg, nil
+}
+
+func setupRelayAddr(cfg *Config) {
+	relayAddr, ok := os.LookupEnv("FLOWS_API_ADDR")
+	if !ok {
+		relayAddr = "0.0.0.0:50051"
```

Comment:
How is `0.0.0.0` a sane default here?

Reply:
Please note: This is taken over from already existing code. Linux treats `0.0.0.0` the same as `localhost`. So let's just use `localhost:50051` here to not break any existing workflow.